pkg/daemon/daemon.go

```diff
@@ -29,6 +31,7 @@ type Daemon interface {
 }
 
 type daemon struct {
+	config     *config.DaemonConfig
```

Comment:
why is that a pointer ? its always initialized as zero struct then populated via its methods

Reply:
changed to be without pointer